core-plugin/src/com/google/gct/idea/debugger/CloudDebuggerRunner.java

```diff
@@ -136,4 +147,44 @@ public void run() {
 
     return debugSession.getRunContentDescriptor();
   }
+
+  private void ensureSingleDebugSession(Project project) throws RunCanceledByUserException {
+
+    // ask to kill background sessions
+    RunManager manager = RunManager.getInstance(project);
```

Comment:
it could help the readability of this code to move the lookup of the of the CloudDebugRunConfig and the CloudDebugProcess into separate methods.